.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
what is the purpose of this file?

Reply:
@artiebits , the .prettierrc file wasn't generated by me. It was generated by Sveltekit, do you still want me to delete it??

Reply:
Also, the .husky/.gitignore was generated by husky itself, I used it's latest versions

Reply:
I would prefer to keep the previous prettier config (default one). It will also make it easier to review your changes because changing code formatting changed every single file

Reply:
hi @kudadam would you be okay to keep the previous prettier formatting? It would help to finalise this PR. Thanks!

Reply:
@artiebits , I can't find the default prettier formatting in your repo. Can you show me where it is so I will fix it in?

Reply:
just remove the existing prettier config file, then Prettier will apply its default config

Reply:
@artiebits , It has been deleted

Reply:
Hey @kudadam. Happy New Year and best wishes for the year ahead!

Since the custom prettier config deleted, will you be able to run Prettier on all files, so the default config applies? 

Sorry for taking so long to reply.

Reply:
@artiebits , Happy New Year to you too, I will start working on it